### ISSUE TITLE
How to set custom translations for this integration

### ISSUE THREAD

**Issue by tobyhage** [shishkin/astro-pagefind#18]
Hello,

With the pagefind searhc library you can set custom translations:

see: https://pagefind.app/docs/ui/#translations

how can i arrange this using this integration

**Comment by shishkin** [comment]
Yes, the props are currently very limited. One would need to add missing props in [here](https://github.com/shishkin/astro-pagefind/blob/main/src/components/Search.d.ts#L1-L5) and pass them into `PagefindUI` in [here](https://github.com/shishkin/astro-pagefind/blob/main/src/components/Search.astro#L25-L28).

Props could just be passed through, but their types still need to be defined for better DX. I've created an [issue upstream](https://github.com/CloudCannon/pagefind/issues/334) to get typings for `PagefindUI`.

**Comment by tobyhage** [comment]
Ok, I will wait for this feature to be implemented. 
Thank you.

**Comment by shishkin** [comment]
For now I've just implemented passing pretty much untyped options to the UI in 1d59d753a3ede5db5715b6d94898bc89701ffbba. It should work for passing translations. Please check out [v1.3.0](https://github.com/shishkin/astro-pagefind/releases/tag/v1.3.0) and let me know if it works for you.
